packages/dynamoose/test/types/tsconfig.json

```diff
@@ -4,7 +4,8 @@
 		"moduleResolution": "node",
 		"lib": ["es2015"],
 		"target": "es2017",
-		"noEmit": true
+		"noEmit": true,
+		"strict": true
```

Comment:
Without `strict: true` (or more specifically `strictNullChecks: true`, The `T | undefined` type simply becomes `T`, rendering these type changes useless. I've mentioned this in the TypeScript documentation above.